Makefile

```diff
@@ -19,6 +19,10 @@ test:
 		go test ./... || failed=$$(($$failed + 1)); \
 		cd ..; \
 	done; \
+	echo "Running frontend tests..."; \
+	cd frontend; \
+	npm run test:unit || failed=$$(($$failed + 1)); \
+	cd ..; \
```

Comment:
To `cd` w sumie niepotrzebne

Reply:
a to nie jest tak że po `make`'u terminal zostanie w katalogu `./frontend`?

Reply:
nie bo makefile izoluje powkłokę dla samego siebie, ale to bez znaczenia tak naprawdę. Możesz mergować